NOTICE.txt

```diff
@@ -35,7 +35,8 @@ This product contains a derivation of the Tony Stone's 'process_test_files.rb'.
 
 ---
 
-This product contains a derivation of SwiftNIO locks.
+This product contains a derivation of SwiftNIO's locks and its build_podspec.sh
+and sanity.sh scripts.
```

Comment:
And generate_linux_tests.rb I guess, do we have to list them all explicitly?

Reply:
good point, I'll just say locks & scripts

Reply:
done